units/src/amount.rs

```diff
@@ -1078,6 +1084,17 @@ impl Amount {
     }
 }
 
+#[cfg(feature = "proptest")]
+impl Arbitrary for Amount {
+    type Parameters = <u64 as Arbitrary>::Parameters;
+    type Strategy = Map<<u64 as Arbitrary>::Strategy, fn(u64) -> Amount>;
+
+    #[inline(always)]
```

Comment:
Why `always`?

Reply:
Shouldn't need `always`. The real optimizations are in inlining the *called* functions.

Reply:
There's a run-time performance increase by always including this in the calling crate.  Since this is an optional dependency I think always inlining is worthwhile.

Reply:
> Shouldn't need always. The real optimizations are in inlining the called functions.

Oops I missed this comment, I think the page hadn't been updated when I replied to tobin.  I actually don't know a lot about the compiler semantics here so if others want to remove always then that's fine.